src/index.ts

```diff
@@ -65,16 +68,21 @@ export const cliArDriveFactory = ({
 		wallet,
 		arfsDao,
 		communityOracle,
-		priceEstimator
+		priceEstimator,
+		shouldBundle,
+		arFSTagSettings,
+		uploadPlanner
 	});
 
 export const cliArDriveAnonymousFactory = ({
 	appName = CLI_APP_NAME,
 	appVersion = CLI_APP_VERSION,
-	arweave = cliArweave
+	arweave = cliArweave,
+	arFSTagSettings
```

Comment:
Remind me why the anonymous ardrive needs tags settings?

Reply:
It does not actually seem to ever use them. But it has had appName/appVersion in its constructor from the start, so I added to maintain the tag availability there

Reply:
I'm backtracking on being ok with this. I think we should remove the tag settings from the anon constructor in core and deprecate the tags in the anon constructor and the non-anon constructor. Make the tag settings a member of the non-anon class. Let's chat.